openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -268,3 +268,14 @@ def test_registration_webinar_admin_delete_permission():
     Test that admins do not have permission to delete webinar registrations
     """
     assert not WebinarRegistrationAdmin.has_delete_permission('self', Mock())
+
+
+@pytest.mark.django_db
+def test_get_deleted_objects_webinar_admin(webinar_admin_instance, request, webinar):
+    """
+    Test get deleted objects for webinar admin is called to bypass permission
```

Comment:
Docstring is misleading. You are not testing if `get_deleted_objects` is called or not, you are explicitly calling it. It can be replaced with:

"Test that `get_deleted_objects` for webinar admin returns an empty set of permissions"